### ISSUE TITLE
[Migrations] Errormessage when executing migration commands

### ISSUE THREAD

**Issue by fashxp** [pimcore/pimcore#8694]
```
In AbstractManagerRegistry.php line 150:
                                                 
  Doctrine ORM Manager named "" does not exist.  
```

fixing doctrine-migrations-bundle version to `doctrine/doctrine-migrations-bundle:3.0.3` solves the problem temporarily.

**Comment by brusch** [comment]
Can be closed as soon as https://github.com/doctrine/DoctrineMigrationsBundle/pull/415 is merged & released.

**Comment by brusch** [comment]
https://github.com/doctrine/DoctrineMigrationsBundle/pull/415 was merged and `3.1.1` released. Done.
